validations/kyc.test.ts

```diff
@@ -6,11 +6,17 @@ import { KycSchema, KycStatus, Network } from '@fiatconnect/fiatconnect-types'
 import path from 'path'
 import { chaiPlugin } from 'api-contract-validator'
 import { MOCK_KYC } from '../src/mock-data/kyc'
+import axios from 'axios'
+import {
+  VALORA_BASE_URL,
+  VALORA_WEBHOOK_HISTORY_ENDPOINT,
+} from '../src/constants'
 
 const apiDefinitionsPath = path.join(config.openapiSpec)
 use(chaiPlugin({ apiDefinitionsPath }))
 
 describe('/kyc', () => {
+  jest.setTimeout(10000)
```

Comment:
doing this affects all tests in the file https://jestjs.io/docs/jest-object#jestsettimeouttimeout

might be better to just increase the timeout for the test on webhooks https://jestjs.io/docs/api#testname-fn-timeout

Reply:
also, 10 seconds might be a little tight for the webhooks tests